docs/contents/installation.md

```diff
@@ -224,7 +225,7 @@ Note that muTect does not provide an easy way to query for the current version,
 
 bcbio-nextgen provides a wrapper around external tools and data, so the actual tools used drive the system requirements. For small projects, it should install on workstations or laptops with a couple GB of memory, and then scale as needed on clusters or multicore machines.
 
-Disk space requirements for the tools, including all system packages are under 4GB. Biological data requirements will depend on the genomes and aligner indices used, but a suggested install with GRCh37 and bowtie/bwa2 indexes uses approximately 35GB of storage during preparation and ~25GB after:
+Disk space requirement for the tools, including all system packages is about 22GB (or more, depending on the type of the file system). Biological data requirements will depend on the genomes and aligner indices used, but a suggested install with GRCh37 and bowtie/bwa2 indexes uses approximately 35GB of storage during preparation and ~25GB after:
```

Comment:
Does this 22GB figure include also the data? As far as I can read from the old text, it only mentioned the actual programs without any data files (but I'm not sure if it is still the same now).

Reply:
This is after `python bcbio_nextgen_install.py [bcbio_path] --tooldir=[bcbio_tools_path] --nodata`.